src/Device.Net/Windows/ApiService.cs

```diff
@@ -14,17 +15,36 @@ namespace Device.Net.Windows
     internal class ApiService : IApiService
     {
         #region Fields
-#if NETFRAMEWORK
-        private const uint FILE_FLAG_OVERLAPPED = 0;
-#else
         private const uint FILE_FLAG_OVERLAPPED = 0x40000000;
-#endif
+
+        private readonly uint fileFlag;
 
         protected ILogger Logger { get; }
         #endregion
 
         #region Constructor
-        public ApiService(ILogger logger = null) => Logger = logger ?? NullLogger.Instance;
+        public ApiService(ILogger logger = null)
```

Comment:
@cupsos I can see what this is about. I see the documentation from Microsoft. 

Unfortunately, this is the wrong place to put this code. This service must remain dumb with no logic. We will need to pass the `fileFlag` in as part of the `CreateConnection()` method.